README.md

```diff
@@ -30,7 +30,20 @@ resources:
 
 To use the templates in your pipeline, see the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops) on templates.
 
-## Versioning & Setup
+Some of the templates in this repository assume that the Azure DevOps Pipelines project is set up a certain way to make use of some of the variables. The required variables are described below, and should be added as Pipeline [Variable Groups](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/variable-groups?view=azure-devops&tabs=yaml).
+
+### Azure Authentication
+
+The Azure authentication assumes that a [Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals) has been set up for the project with all the required permissions.
```

Comment:
It might be useful to say what exactly the required permissions might be?

Reply:
Done. Although it kind of depends on the context of the project, so hopefully I've made that a bit clearer?